src/client/Microsoft.Identity.Client/Http/LinearRetryPolicy.cs

```diff
@@ -23,6 +25,9 @@ public LinearRetryPolicy(int delayMilliseconds, int maxRetries, Func<HttpRespons
 
         public bool pauseForRetry(HttpResponse response, Exception exception, int retryCount)
```

Comment:
nit: please rename to `PauseForRetry` as this violates naming convention.

Reply:
done